internal/backend/discussion.go

```diff
@@ -439,6 +464,9 @@ func updateDiscussionObj(disc *model.Discussion, input model.DiscussionInput) {
 	if input.LastPostCreatedAt != nil {
 		disc.LastPostCreatedAt = input.LastPostCreatedAt
 	}
+	if input.ShuffleID != nil {
```

Comment:
I removed this as this is going to be handled by an increment sql query instead.